ci/circle

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env python2.7
```

Comment:
I don't feel strongly but not a fan of capitals in directories in general and I haven't seen the "CI" convention before.

anyway, not a problem, I will rename